library/panic_abort/src/android.rs

```diff
@@ -0,0 +1,52 @@
+#![feature(std_internals)]
+#![feature(try_reserve)]
+
+use alloc::string::String;
+use alloc::vec::Vec;
+use core::mem::transmute;
+use core::panic::BoxMeUp;
+
+const ANDROID_SET_ABORT_MESSAGE: &[u8] = b"android_set_abort_message\0";
+type set_abort_message_type = unsafe extern "C" fn(*const libc::c_char) -> ();
```

Comment:
Doesn't this produce a warning about not using CamelCase for the type alias?

Reply:
I did not run `x.py test tidy` before. Done.